R/git.R

```diff
@@ -0,0 +1,34 @@
+#' Git information
+#' 
+#' Displays git information from the repository of the current working space.
+#' 
+#' @import git2r
+#' @import shiny
+#' @import glue
+#' @export
+git_info <- function() {
+  repo <- tryCatch({
+            repository(".")
+          }, error = function(e) {
+             NULL
+          })
+  if (is.null(repo)) {
+    git_message <- "It looks like you're not in a valid git repo."
+  } else {
+    branch <- repository_head(repo)[[1]]
```

Comment:
`repository_head` function is available only from 0.22.1 git2r version. Please add in DESCRIPTION:

```
Imports:
    git2r (>= 0.22.1)
```

Reply:
Thanks for spotting that. Fixed.